src/main/java/org/opensearch/knn/index/util/KNNLibrary.java

```diff
@@ -138,6 +139,20 @@
      */
     Map<String, Object> getMethodAsMap(KNNMethodContext knnMethodContext);
 
+    /**
+     * Getter for initialized
+     *
+     * @return whether library has been initialized
+     */
+    Boolean isInitialized();
```

Comment:
why not isLoaded?

Reply:
Is loaded implies it can be unloaded. I thought initialized was better.

Reply:
Got it.